rbenchmarker/benchmarker/README.md

```diff
@@ -0,0 +1,45 @@
+# Generating test data
+
+The test data can be generated by building the docker image defined and running it:
```

Comment:
I don't think this is creating test data?

Reply:
corrected